File: bmi.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv)

{
    
int masa = 0;
float wzrost = 0;
float bmi = 0;
 
 
 cout << "Podaj masę (kg): ";
 cin >> masa;
 
while(masa < 0 or masa > 1000)
{
   cout << "Błędna wartosć ";
   cin >> masa;
}
 
 cout << "Podaj wzrost (cm): ";
cin >> wzrost;
 
while(wzrost < 0 or wzrost > 300)
{
   cout << "Błedna wartość";
   cin >> wzrost;
}
 
 
bmi = masa / (wzrost * wzrost ); 
 
 cout << "Twoje BMI wynosi:" << endl;
cout << bmi << endl << endl;
 
 
if (bmi < 18.5)
{
 cout << "niedowaga" << endl;
}
 
if (bmi > 18.5 and bmi < 24.9)
{
 cout << "norma" << endl;
}
 
if (bmi > 25)
{
 cout << "nadwaga" << endl;
}

if (bmi > 30)
{
 cout << "otyłość" << endl;
}

return 0;
}

File: bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    masa = float(input("podaj masę(kg)"))
    print(masa)
    wzrost = float(input("podaj wzrost(m)"))
    print(wzrost)

    c = masa / (wzrost * wzrost)
    print ("bmi:", c)

    if c < 18.5:
        print("niedowaga")
    elif c <= 24.9:
        print("norma")
    elif c >= 25:
        print("nadwaga")
    elif c >= 30:
        print("otyłość")



    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
